docs/semantic-versioning-policy.md

```diff
@@ -0,0 +1,27 @@
+# Semantic Versioning Policy
+
+`eslint-plugin-testing-library` follows [Semantic Versioning](https://semver.org/). However, for [the same reason as ESLint itself](https://github.com/eslint/eslint#semantic-versioning-policy), it's not always clear when a minor or major version bump occurs. To help clarify this for everyone, we've defined the following Semantic Versioning Policy for this ESLint plugin:
+
+- Patch release (intended to not break your lint build)
+  - A bug fix in a rule that results in `eslint-plugin-testing-library` reporting fewer errors.
+  - A bug fix to the core.
+  - Re-releasing after a failed release (i.e., publishing a release that doesn't work for anyone).
+- Minor release (might break your lint build)
+  - A bug fix in a rule that results in `eslint-plugin-testing-library` reporting more errors.
+  - A new rule is created.
+  - A new option to an existing rule that does not result in ESLint reporting more errors by default.
+  - A new option to an existing rule that might result in ESLint reporting more errors by default.
```

Comment:
As discussed in #353, this is the main difference from ESLint SemVer Policy so it's a minor instead of a major.